frontend/public/src/components/CourseInfoBox.js

```diff
@@ -21,6 +21,18 @@ type CourseInfoBoxProps = {
   toggleUpgradeDialogVisibility: () => Promise<any>,
   setCurrentCourseRun: (run: EnrollmentFlaggedCourseRun) => Promise<any>
 }
+const getCourseDates = (run, isMoreDates = false) => {
+  let startDate = formatPrettyDate(parseDateString(run.start_date))
+  if (run.is_self_paced && moment(run.start_date).isBefore(moment())) {
+    startDate = "Anytime"
+  }
+  return (
+    <>
+      <b>Start:</b> {startDate} {isMoreDates ? null : <br />}
+      <b>End:</b> {formatPrettyDate(parseDateString(run.end_date))}
```

Comment:
This line is throwing javascript errors when `end_date` is not defined for the courserun.